crates/ruff/src/rules/ruff/snapshots/ruff__rules__ruff__tests__RUF200_invalid_author.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: crates/ruff/src/rules/ruff/mod.rs
+---
+./resources/test/fixtures/ruff/pyproject_toml/invalid_author/pyproject.toml:6:30: RUF200 Failed to parse pyproject.toml: invalid type: integer `1`, expected a string
+  |
+6 | # Ensure that the spans from toml handle utf-8 correctly
+7 | authors = [
+8 |     { name = "콘스턴틴", email = 1 }
+  |                                  ^ RUF200
```

Comment:
i'm afraid the korean font isn't monospace, it looks at least a bit better on my machine (but the columns are still slightly misaligned)

![image](https://github.com/charliermarsh/ruff/assets/6826232/a3fbc620-f5ad-476c-ab83-c2237b5ce914)

Reply:
opened https://github.com/toml-rs/toml/issues/556

Reply:
it's byte offsets indeed, confused why it's 30 chars in line 8 and 34 chars whitespaces below

Reply:
Apparently the width of hangul syllables is 2, but for historic reasons there no proper hangul monospace. I've replaced the test string with something else

![image](https://github.com/charliermarsh/ruff/assets/6826232/181b1a9b-b511-4271-a4d8-be10143514a4)